hw/top_earlgrey/data/ip/chip_rv_dm_testplan.hjson

```diff
@@ -0,0 +1,289 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: rv_dm
+  testpoints: [
+    // RV_DM (JTAG) tests:
+    {
+      name: chip_jtag_csr_rw
+      desc: '''
+            Verify accessibility of all the CSRs in the chip over JTAG.
+
+            - Shuffle the list of CSRs first to remove the effect of ordering.
+            - Write all CSRs via JTAG interface with a random value.
+            - Shuffle the list of CSRs yet again.
+            - Read all CSRs back and check their values for correctness while adhering to the CSR's
+              access policies.
+            - Accesses to CSRs external to `rv_dm` go through RV_DM SBA interface into the `xbar`.
+            '''
+      stage: V2
+      si_stage: SV2
+      tests: ["chip_jtag_csr_rw"]
+      lc_states: ["TEST_UNLOCKED", "DEV", "RMA"]
+      host_support: "true"
```

Comment:
I have not seen `host_support` before, what does it mean exactly? Is there a place where the valid attribute are defined?

Reply:
It means the test will require a component that's outside of the device itself to run. It's defined in this document: https://docs.google.com/document/d/14b1RWc2jozg9CfuFK4xCUMR5j4gCjrgYrif40Irqc7Q/edit#heading=h.4cj1geh4tmef